### PR TITLE
chore: pass SinkReplyBuilder and Transaction explicitly. Part2

### PR DESCRIPTION
**Pull request by romange**
This pr follows #3946

### CODE REVIEW

File: src/server/generic_family.h [hunk 1]
```diff
@@ -38,42 +32,40 @@ class GenericFamily {
...
-  static void Select(CmdArgList args, ConnectionContext* cntx);
-  static void Scan(CmdArgList args, ConnectionContext* cntx);
-  static void Time(CmdArgList args, ConnectionContext* cntx);
-  static void Type(CmdArgList args, ConnectionContext* cntx);
-  static void Dump(CmdArgList args, ConnectionContext* cntx);
-  static void Restore(CmdArgList args, ConnectionContext* cntx);
-  static void RandomKey(CmdArgList args, ConnectionContext* cntx);
-  static void FieldTtl(CmdArgList args, ConnectionContext* cntx);
-  static void FieldExpire(CmdArgList args, ConnectionContext* cntx);
+  static void Del(CmdArgList args, Transaction* tx, SinkReplyBuilder* builder);
```
Comment (romange) [inline src/server/generic_family.h:37]: now we are going to pass `Transaction* tx, SinkReplyBuilder* builder` explicitly.

File: src/server/generic_family.cc [hunk 2]
```diff
@@ -863,6 +862,99 @@ OpResult<uint64_t> OpTtl(Transaction* t, EngineShard* shard, string_view key) {
   }
 }
 
+ErrorReply RenameGeneric(CmdArgList args, bool destination_should_not_exist, Transaction* tx) {
```
Comment (romange) [inline src/server/generic_family.cc:865]: moved from below.

**Comment by romange** [comment]
@BorysTheDev  ping

File: src/server/db_slice.cc [hunk 3]
```diff
@@ -13,7 +13,6 @@
 #include "server/cluster/cluster_defs.h"
 #include "server/engine_shard_set.h"
 #include "server/error.h"
-#include "server/generic_family.h"
```
Comment (BorysTheDev) [inline src/server/db_slice.cc:16]: I like when we remove such huge files from includes

File: src/server/generic_family.cc [hunk 4]
```diff
@@ -863,6 +863,99 @@ OpResult<uint64_t> OpTtl(Transaction* t, EngineShard* shard, string_view key) {
...
+    } else if (arg_sv == "XX") {
+      flags |= ExpireFlags::EXPIRE_XX;
+    } else if (arg_sv == "GT") {
+      flags |= ExpireFlags::EXPIRE_GT;
+    } else if (arg_sv == "LT") {
+      flags |= ExpireFlags::EXPIRE_LT;
+    } else {
+      builder->SendError(absl::StrCat("Unsupported option: ", arg_sv));
+      return nullopt;
+    }
```
Comment (BorysTheDev) [inline src/server/generic_family.cc:946]: can you rewrite it with CmdArgParser

Comment (romange) [reply src/server/generic_family.cc:946]: I prefer to avoid rewriting stuff during refactoring PRs. this one is huge as it is.

**Review by BorysTheDev — APPROVED** [review]

**Pull request merged**